src/webpages/Articles.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import ArticleCard from "../components/ArticleCard";
+import api from "../api";
+
+export default function Articles() {
+  const [articles, setArticles] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    console.log("Articles component mounted");
```

Comment:
Don't forget to remove console.logs before merging once they've served their purpose for debugging, etc. Want to avoid having them clogging up the console in production mode.